Governance documentation (#711)
AstarNetwork/astar-docs: docs/use/governance/subsquare_guide.md

---

After submitting the proposal, some time needs to pass before the transaction is confirmed and finalized.

<Figure caption="Public Proposal - 4" src={require('/docs/use/img/18_Subsquare_public_proposal/public_proposal_3.png').default } width="100%" />

---

Afterwards, it can be observed under the list of public proposals.
The status will be marked as `Proposed`, since it hasn't been _tabled_ (or _canceled_) yet.

<Figure caption="Public Proposal - 5" src={require('/docs/use/img/18_Subsquare_public_proposal/public_proposal_4.png').default } width="100%" />

---

Clicking on the proposal will display more details, and allow to take further actions.

<Figure caption="Public Proposal - 6" src={require('/docs/use/img/18_Subsquare_public_proposal/public_proposal_5.png').default } width="100%" />

---

Proposals can be edited by clicking on the `Edit` button. Title and description can be changed, even  a graphica image can be uploaded.
Users can discuss the proposal in the comments section.

<Figure caption="Public Proposal - 7" src={require('/docs/use/img/18_Subsquare_public_proposal/public_proposal_6.png').default } width="100%" />

---

> Clicking on the `Second` button will allow the user to second the proposal, increasing the chances of it being tabled and upgraded to a referendum.

Reviewer: ```suggestion
Clicking the `Second` button to support the proposal, increase its chances of being tabled and upgraded to a referendum.
```

It is possible to second a proposal multiple times.

<Figure caption="Public Proposal - 8" src={require('/docs/use/img/18_Subsquare_public_proposal/public_proposal_7.png').default } width="100%" />

### Treasury

Clicking on the `Treasury` tab on the sidebar will display the current state of the treasury & spending requests.

<Figure caption="Treasury - 1" src={require('/docs/use/img/19_Subsquare_Treasury/treasury_1.png').default } width="100%" />

---

Clicking on the `New Proposal` button will allow the creation of a new spending proposal.
User needs to pick the payout amount & the beneficiary.
The _Proposal bond_ refers to the amount that needs to be deposited in order to create the proposal.
In case the proposal is rejected, the bond will be slashed.
Otherwise, the bond will be refunded.

<Figure caption="Treasury - 2" src={require('/docs/use/img/19_Subsquare_Treasury/treasury_2.png').default } width="100%" />

---

After submitting the proposal, it will be displayed on the treasury page.
The proposal name & description can be edited.

<Figure caption="Treasury - 3" src={require('/docs/use/img/19_Subsquare_Treasury/treasury_3.png').default } width="100%" />

:::note

> The exact same steps can be reused for the community treasury proposals.

Reviewer: ```suggestion
You can reuse the exact same steps for community treasury proposals.
```

:::

### Vote Delegation

Click on the `Delegation` tab on the sidebar to access the vote delegation page.
This is an overview of _delegation_ related actions and statistics.

Clicking on the `+ Delegate` button will allow the delegation of votes to another account.

<Figure caption="Vote Delegation - 1" src={require('/docs/use/img/20_Subsquare_Delegation/delegation_1.png').default } width="100%" />

---

The `Target` is the account to which the votes are being delegated to.
The `Balance` is the amount of native currency we're delegating. This is the amount that will be _locked_.
:::important
The action of delegating voting power **DOES NOT** transfer the ownership of the tokens. They are merely _locked_ for normal use, but remain in the ownership of the delegator.
:::

> The `Conviction` allows us to amplify the voting power being transferred, at the expense of a longer _unlock_ period.

Reviewer: ```suggestion
The `Conviction` setting determines how much your voting power is amplified in exchange for a longer token lock period.
```

<Figure caption="Voting - 4" src={require('/docs/use/img/21_Subsquare_voting/voting_4.png').default } width="100%" />

---

In case a user changes their mind, the vote can be removed by clicking on the `Remove Vote` button.
This won't trigger any _unlocking period_ and the user is free to cast their vote again.

<Figure caption="Voting - 5" src={require('/docs/use/img/21_Subsquare_voting/voting_5.png').default } width="100%" />

---

This is an example of an account that is voting but has some votes delegated to it.
In this particular situation, the account has:

* 100 SBY free balance
* 300 SBY worth of voting power delegated to it.

Although not visible on this image, the account who delegated the votes did it with 100 SBY and conviction set to **3**.

The account votes with 50 SBY, conviction set to **4**.
This means that the total amount of votes for this account will be: **50 SBY x 4 + 300 SBY = 500 SBY**.

<Figure caption="Voting - 6" src={require('/docs/use/img/21_Subsquare_voting/voting_6.png').default } width="100%" />

---

The new total turnout is 200 SBY - 50 SBY from the first `aye` vote, another 50 SBY from the second `aye` vote, and 100 SBY from the delegated vote.

> The total votes toward `aye` is 600 SBY in total - 100 SBY from the first `aye` vote, and another 500 SBY from the second `aye` vote.

Author: I'll omit this since it might look like 600 SBY - 100 SBY 😅
